src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,17 +130,21 @@ public void setPerson(Person target, Person editedPerson) {
      */
     @Override
     public ObservableList<Person> getFilteredPersonList() {
-        return filteredPersons;
+        return sortedPersons;
```

Comment:
this naming is a bit confusing, maybe change it to `sortedAndFilteredPersons`? 🤔